USER_GUIDE.md

```diff
@@ -67,13 +75,12 @@ func example() error {
 	}`)
 
 	// Create an index with non-default settings.
-	createIndex := opensearchapi.IndicesCreateRequest{
+	createIndex := opensearchapi.IndicesCreateReq{
```

Comment:
Do you feel strongly about the rename of `Request` to `Req`? Thinking of someone porting their code, this feels like they will have to do an unnecessary bulk change.

Reply:
I'd prefer the short version to reduce line length as the type names can get extremely long, for example: `opensearchapi.ComponentTemplateCreateReq{}`. The similarity of the new type was not by intention. So there might be a lot of types with similar new names but also a lot with new different type names.  Related to the previously named type, following is the old type name `opensearchapi.ClusterPutComponentTemplateRequest{}`.